### PR TITLE
Components and redux store 

### PR DESCRIPTION
**Pull request by grace-mugoiri**
Kindly review my work for milestone which deals with project structure and actions (reducers) for this section.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/extensions */
```
Comment (JamesOkunlade) [inline src/App.js]: Your `app.js` component that should go into the `src/components/` folder.

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+class Book extends React.Component {
+  render() {
+    const book = this.props.book;
```
Comment (JamesOkunlade) [inline src/components/Book.js]: Destructuring the book props in this case should be `const { book } = this.props.book;`

File: src/containers/BooksForm.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-restricted-syntax */
+import React, { Component } from 'react';
+
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bookGenres: ['Action and Adventure', 'Anthology', 'Chick lit', 'Crime', 'Drama', 'Fantasy'],
```
Comment (JamesOkunlade) [inline src/containers/BooksForm.js]: Your list of categories will be in a normal `const` data type in your `render()` section or imported from a separate `util` file and not in the `state`. What should go into the `state` are `title` and `category` 

```
this.state = {
      title: '',
      category: 'Action',
    };
```

File: src/containers/BooksForm.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    };
+  }
+
+  render() {
+    const { bookGenres } = this.state;
```
Comment (JamesOkunlade) [inline src/containers/BooksForm.js]: Remove this line as your genres will not be coming from the `state`. Implement my above point instead and also write these lines of code to map all the genres to display in the select options:

```
const bookGenres = ['Action and Adventure', 'Anthology', 'Chick lit', 'Crime', 'Drama', 'Fantasy'];

const bookOptions = bookGenres.map(category => <option key={category} value={category}>{category}</option>);

```

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
# ♻️ Changes required

**Hello guys,**

Great job so far 🙂 I have left some comments for you to make corrections on. Please check them out below.

Also, remove all the files that are not needed anymore from your project, like the `App.test.js`, `logo.svg`,  and `setupTests.js` all in your `src` folder and the `.png`s in your `public` folder.

Request for re-review when you're done. 

**Goodluck** 😁  
[James](https://twitter.com/JamesOkunlade)

**Review by bellom — APPROVED** [review]
## Hi @grace-mugoiri   :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**